### DISCUSSION TITLE
MDF_5 (portrait) [5"]

### DISCUSSION THREAD

**Discussion by a-sber-rh** [mathoudebine/turing-smart-screen-python#386] [Themes]
I adapted the Material Dark Full design from [geekm0nkey](https://github.com/geekm0nkey) to fit my 5" screen. I also added the names of the components I have and included the date and time.

The original one it's [here](https://github.com/mathoudebine/turing-smart-screen-python/discussions/284)

![preview](https://github.com/mathoudebine/turing-smart-screen-python/assets/149914394/8a99d11d-8d45-47e6-ac52-1e7f9fe1f7d1)

[MDF_5.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/13258091/MDF_5.zip)

**Comment by ooemir23** [comment]
Very nicee. Thx bro

**Comment by joyzao** [comment]
Hello, how can I monitor the fps? I already have the Riva tuner

**Reply by aStroNuts04** [reply]
Hi have you found a fix for this? Riva dosent do anything for me

**Comment by aStroNuts04** [comment]
Hi how did you get the fps to work?

**Comment by bandolero30** [comment]
thank you!
